### PR TITLE
Adding sequence diagrams for bulk Quotes

### PR DESCRIPTION
**Pull request by elnyry-sam-k**
Adding sequence diagrams for
- POST /bulkQuotes
- PUT /bulkQuotes/{ID}
- PUT /bulkQuotes/{ID}/error
- GET /bulkQuotes

Posted snapshots below to help with reviews

### CODE REVIEW

File: mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-post-bulk-quotes-2.2.0.plantuml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+end
+|||
+Switch -> Switch: Use fspiop-destination header to retrieve\n bulk quotes endpoint for Payee DFSP
+alt Payee bulkQuotes endpoint found
+    note right of Switch: Switch forwards bulk quote request to Payee FSP
+    Switch -\ PayeeFSP: POST /bulkQuotes
+    Switch \-- PayeeFSP: 202 OK
+
+    PayeeFSP -> PayeeFSP: Payee FSP calculates individual quotes\nand responds with a bulk Quote result
+    alt Payee bulkQuotes processing successful
```
Comment (oderayi) [inline mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-post-bulk-quotes-2.2.0.plantuml:67]:
```suggestion
    alt Payee bulk quotes processing successful
```

File: mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-post-bulk-quotes-2.2.0.plantuml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    end note
+    PayerFSP /- Switch: PUT /bulkQuotes/{ID}/error
+    PayerFSP --/ Switch: 200 OK
+|||
+else Request is a duplicate and a resend
+    Switch -> Switch: Switch handles resend scenario
+end
+|||
+Switch -> Switch: Use fspiop-destination header to retrieve\n bulk quotes endpoint for Payee DFSP
+alt Payee bulkQuotes endpoint found
```
Comment (oderayi) [inline mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-post-bulk-quotes-2.2.0.plantuml:61]:
```suggestion
alt Payee bulk quotes endpoint found
```

File: mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-post-bulk-quotes-2.2.0.plantuml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            Switch \-- PayeeFSP: 200 OK
+        end
+        alt Response is a duplicate and a resend
+            Switch -> Switch: Switch handles resend scenario
+        end
+
+        note left of Switch: Switch forwards quote response to Payer FSP
+        PayerFSP /- Switch: PUT /bulkQuotes/{ID}
+        PayerFSP --/ Switch: 200 OK
+    else Payee rejects bulk quote or encounters and error
```
Comment (oderayi) [inline mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-post-bulk-quotes-2.2.0.plantuml:85]:
```suggestion
    else Payee rejects bulk quote or encounters an error
```

File: mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-quotes-overview-1.0.0.plantuml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    alt Payee FSP successfully calculates quote
+    
+        note over "Payee FSP", Switch: Payee FSP responds to quote request
+        "Payee FSP" -\ Switch: PUT /quotes/{ID}
+        Switch --/ "Payee FSP": 200 Ok
+        
+        Switch -> Switch: Validate Quote Response
+        
+        alt response is ok
+        ƒ
```
Comment (oderayi) [inline mojaloop-technical-overview/quoting-service/assets/diagrams/sequence/seq-quotes-overview-1.0.0.plantuml:30]:
```suggestion
```

**Comment by elnyry-sam-k** [comment]
seq-post-bulk-quotes-2 2 0

**Comment by elnyry-sam-k** [comment]
seq-get-bulk-quotes-2 1 0

**Review by simeonoriko — APPROVED** [review]

**Pull request merged**